libs/indibase/connectionplugins/connectionserial.h

```diff
@@ -42,7 +42,7 @@ class Serial : public Interface
      * \brief Supported baud rates
      * \note: Default baud rate is 9600. To change default baud rate, use setDefaultBaudrate(..) function.
      */
-    typedef enum { B_9600, B_19200, B_38400, B_57600, B_115200, B_230400 } BaudRate;
+    typedef enum { B_9600, B_19200, B_38400, B_57600, B_115200, B_230400, B_460800, B_576000, B_921600 } BaudRate;
```

Comment:
Ok hopefully these baud rates would be OK for MacOS.

Reply:
Yes, but I just want to be sure, unfortunately these were changed for tests and forgot to not include them into the commit.